kitchensink/src/app/common/PageElements/FoundationElements/ColorRow/ColorRow.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ColorBox from '../ColorBox/ColorBox';
+import './colorRow.css';
+
+const ColorRow = (props) => (
+  <div styleName="colorRowContainer">
+    <table>
+      {
+        props.showInfo
+        && (
+          <thead>
+            <tr>
+              {
+                props.title.length > 0 && <th key="base-col-header" style={{ width: (props.rowHeader.width ? props.rowHeader.width : 'auto') }} styleName={classNames('title', props.rowSize, 'base-column')}>{props.title}</th>
+              }
+              {
+                props.preColumns > 0
+                && <th colSpan={props.preColumns}>&nbsp;</th>
+              }
+              {
+                (props.rowHeader && Object.keys(props.rowHeader).length > 0)
+                && props.rowData.map((element) => {
+                  if (props.showInfo) {
+                    const title = element.title || '';
+                    return (
+                      <th key={`${element.colors[0].name}-title`}>
+                        <div styleName={classNames('title', props.rowSize)}>{title}</div>
+                      </th>
+                    );
+                  }
+                  return null;
+                })
+              }
+            </tr>
+          </thead>
+        )
+      }
+      <tbody>
+        <tr styleName="colorRow">
+          {
+            (props.rowHeader && props.rowHeader.colors)
+            && (
+              <td key="base-row-header" styleName={classNames('rowHeader', props.rowSize)}>
+                <ColorBox
+                  variableName={props.rowHeader.colors[0].name}
+                  hexValue={props.rowHeader.colors[0].hex}
+                  size={props.rowSize}
+                  darkFont={props.rowHeader.colors[0].darkFont}
+                  bordered={props.rowHeader.colors[0].bordered}
+                  stackPosition={props.rowHeader.colors[0].stack}
+                  infoBox={{ infoTitle: props.rowHeader.colors[0].colorName, infoLabel: props.rowHeader.colors[0].infoLabel }}
+                  key={props.rowHeader.colors[0].name}
```

Comment:
I think you do not need the `key` props here `key={props.rowHeader.colors[0].name}`